drivers/rn2xx3/rn2xx3.c

```diff
@@ -175,8 +192,10 @@ int rn2xx3_init(rn2xx3_t *dev)
 
 int rn2xx3_sys_reset(rn2xx3_t *dev)
 {
-    memset(dev->cmd_buf, 0, strlen(dev->cmd_buf));
-    snprintf(dev->cmd_buf, 10, "sys reset");
+    size_t p = 0;
+    dev->cmd_buf[p] = 0;
```

Comment:
This statement is useless. It sets the first char in the array to zero. And the snprintf that follows will always write one or more chars to the start of the array.

Note that the same statement after snprintf is essential, and must stay.